README.md

```diff
@@ -409,6 +413,88 @@ object.
     </tbody>
 </table>
 
+#### `gen-test-runner` Config
+All configuration options for the `gen-test-runner` task are found on the `config.genTestRunner`
+object.
+
+##### `GenTestRunner`
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>configs</code></td>
+            <td><code>List&lt;TestRunnerConfig&gt;</code></td>
+            <td><code>[TestRunnerConfig()]</code></td>
+            <td>The list of runner configurations used to create individual test runners</td>
+        </tr>
+    </tbody>
+</table>
+
+##### `TestRunnerConfig`
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>additionalImports</code></td>
+            <td><code>List&lt;String&gt;</code></td>
+            <td><code>[]</code></td>
+            <td>Any other imports necessary to have in the generated runner file</td>
+        </tr>      
+        <tr>
+            <td><code>commandsPriorToTesting</code></td>
```

Comment:
not sure I like this naming.  how about `preTestCommands`?  any other suggestions?